tests/unit/yarn_slider/test_help.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from mock import patch
+import os
+
+import prestoadmin
+
+from tests.unit.test_main import BaseMainCase
+
+
+#
+# Getting this and TestStandaloneHelp running, and subsequently running
```

Comment:
Could you make this comment briefer (maybe just paragraphs 2 and 4)?  It's a bit overwhelming.